domains.txt

```diff
@@ -581,26 +642,33 @@ rightfutures.com
 rjcgroup.co.uk
 rmirecruitment.com
 rmw-uk.com
+roberthalf.co.uk
 robertwalters.com
 roc-search.com
+rothsteins.co.uk
 rpint.com
 rtrs.co.uk
 rubeus-solutions.com
+rubiconconsulting.co.uk
 rullion.co.uk
 s-sa.co.uk
 sabre-rm.com
 salesjobsoffshore.co.uk
 sandersonplc-update.com
 sandersonplc.com
+saragossa.co.uk
```

Comment:
our role is to match clients with high calibre candidates in financial technology, senior appointments and data science.